kinto/core/authorization.py

```diff
@@ -234,7 +234,8 @@ def _find_required_permission(self, request, service):
         required_permission = self.method_permissions.get(method)
 
         # For create permission, the object id is the plural endpoint.
-        collection_path = service.collection_path.format(**request.matchdict)
+        plural_endpoint = service.collection_path.decode('utf-8')
```

Comment:
This won't work with Python 3 because it will already be a string. In that case we need to use decode_string from utils.